nodejs/week2/Node.js-week2-homework/meal-sharing.app/src/backend/routes/meals.js

```diff
@@ -0,0 +1,62 @@
+const express = require("express");
+const router = express.Router();
+
+const meals = require("../data/meals.json");
+
+router.get("/meals", (request, response) => {
+	//meals
+	if (request.url === "/meals") {
+		response.json(meals);
+	}
```

Comment:
I think it is better to change condition, that sends array of meals, so it will not depend on route name. 
(Otherwise you always should think about changing it, if route name is changed.)
Fx, you can check if req.query is an empty object.
```
if (Object.keys(request.query).length === 0 && request.query.constructor === Object) {
		response.json(meals);
	}
```
Maybe you can find a better way https://stackoverflow.com/questions/679915/how-do-i-test-for-an-empty-javascript-object

Reply:
okay, done that and thanks for the link